src/reducers/sources.js

```diff
@@ -139,8 +141,12 @@ function update(
     case "NAVIGATE":
       const source = getSelectedSource({ sources: state });
       const url = source && source.get("url");
-      prefs.pendingSelectedLocation = { url };
-      return State().set("pendingSelectedLocation", { url });
+
+      if (!url) {
+        return initialState();
+      }
```

Comment:
this is a race condition where we don't have a selected source, but we had one prior... It's a small fix that came from some fast tests.